lib/pangea/models/practice_activities.dart/practice_activity_model.dart

```diff
@@ -12,16 +12,28 @@ import 'package:sentry_flutter/sentry_flutter.dart';
 class ConstructIdentifier {
   final String lemma;
   final ConstructTypeEnum type;
+  String? category;
 
-  ConstructIdentifier({required this.lemma, required this.type});
+  ConstructIdentifier({
+    required this.lemma,
+    required this.type,
+    this.category,
+  });
 
   factory ConstructIdentifier.fromJson(Map<String, dynamic> json) {
+    final categoryEntry = json['cat'];
```

Comment:
I think we want json['cat'] ?? json ['categories'] for backwards compatibility here.

Reply:
Need to add to the ContructIdentifier toJson as well!